docs/reference/commandline/build.md

```diff
@@ -29,7 +29,7 @@ Build an image from a Dockerfile
 | `--label`                           | `list`        |           | Set metadata for an image                                         |
 | `-m`, `--memory`                    | `bytes`       | `0`       | Memory limit                                                      |
 | `--memory-swap`                     | `bytes`       | `0`       | Swap limit equal to memory plus swap: -1 to enable unlimited swap |
-| `--network`                         | `string`      | `default` | Set the networking mode for the RUN instructions during build     |
+| `--network`                         | `string`      | `default` | Set [the networking mode](https://docs.docker.com/network/) for the RUN instructions during build     |
```

Comment:
This table is actually generated from the source code (I thought we had a verify step for this one to check it, but looks like it doesn't flag this 🤔). The table in _markdown_ is only used on GitHub (and generated from the YAML properties on docs.docker.com).

What I think we need here instead is to either add a section for this  option further down (having the right anchor tag to match the flag name), and add the link for details in there, or perhaps we need to add a link to the `buildx build` documentation as alternative link (if that page _does_ have a description).

Reply:
Hmm, there don't actually appear to be many sections for more explanation in this doc. Is there any reason we can't add links to code comments? Or we'd just rather not?